numeric.c

```diff
@@ -76,18 +76,36 @@ S_strtod(pTHX_ const char * const s, char ** e)
 
 /*
 
-=for apidoc my_strtod
+=for apidoc      my_strtod
+=for apidoc_item Strtod
 
-This function is equivalent to the libc strtod() function, and is available
-even on platforms that lack plain strtod().  Its return value is the best
-available precision depending on platform capabilities and F<Configure>
-options.
+These are identical.
+
+They act like the libc C<L<strtod(3)>> function, with three exceptions:
+
+=over
+
+=item 1.
+
+Their return value is an NV.  Plain C<strod> returns a double precision value.
+NV's are always the largest precision available, which depends on platform
+capabilities and F<Configure> options.
```

Comment:
NV isn't the largest precision enabled, otherwise it would always be __float128 on gcc.

It's just the floating point type perl is configured to use.